Submission.java

```diff
@@ -464,7 +464,7 @@ public int getSoundBlocksForStage()
      * @param scriptName 
      * @return category, null if no mapping
      */
-    private String getCategory(String scriptName)
+    protected static String getCategory(String scriptName)
     {
```

Comment:
This is probably a dumb question, but what is the reasoning for changing this method to protected?  Is it inherited somewhere?